statstracker/src/tests/App.test.js

```diff
@@ -0,0 +1,117 @@
+// Required to get <App /> running:
+import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from '../reducers/reducers';
+import App from '../components/App';
+const store = createStore(rootReducer);
+
+import { render, fireEvent } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+const packagedApp = (
+  <Provider store={store}>
+    <App />
+  </Provider>
+);
+
+const { getByText } = render(packagedApp);
+
+// I think this is a pretty good way to click different buttons any number of times:
+const click = (button, number = 1) => {
+  // Wow! I finally used RegExp in a real project. This was hard for me to figure out! Lots of trial and error and reading...
+  const regex = new RegExp(button.concat('$'), 'i');
+  for (let i = 0; i < number; i += 1) {
+    fireEvent.click(getByText(regex));
+  }
```

Comment:
very neat function! regex and RegExp classes are tough at first but very powerful once you get used to them--if you want more modular practice aside from just looking at docs/cheatsheets, the Freecodecamp portion of courses dealing with regex can be very useful